CHANGELOG.md

```diff
@@ -25,7 +25,7 @@ the detailed section referring to by linking pull requests or issues.
 - Assets Page search input field is now case insensitive
 - Enchance EDC UI terminologies
 - Performance improvement when fetching a single contract agreement
-- Temporary reimplement the Edit Asset as a Dialog
+- Temporary reimplement the Create Asset as a Dialog
```

Comment:
<img width="2560" alt="image" src="https://github.com/user-attachments/assets/6843a163-8563-498c-96d3-436c3e1e4502">

for edit case you no longer have the three options for on request data offers